NetworkPkg/UefiPxeBcDxe/PxeBcDriver.c

```diff
@@ -1641,7 +1641,7 @@ PxeBcStop (
            &gEfiCallerIdGuid,
            &Private->Id
            );
-    FreePool (Private);
+    Private->DeviceDisconnected = TRUE;          // MU_CHANGE - 162958

```

Comment:
For these two can you leave the old code in, commented out? That is our usual style.